spec/features/user_signs_up_spec.rb

```diff
@@ -36,4 +37,19 @@
     expect(page).to have_content("can't be blank")
     expect(page).to_not have_content('Sign Out')
   end
+
+  xscenario "user uploads a profile photo" do
```

Comment:
this test should not be pending and is a requirement to merging in this feature. Is something wrong with the test that I can help with?

Reply:
Nothing is wrong, we are just done for the day. 
We put the pending test in as a place saver to help remind us that we have to actually write a test. We will write the test tomorrow.